### PR TITLE
:seedling: Change KubeConfigSecret to a struct type.

### PR DESCRIPTION
**Pull request by xuezhaojun**
## Summary

The KubeConfigSecret should not only be string but a struct in case we want to add addtional information such as `context` in the future.

### CODE REVIEW

File: operator/v1/types_klusterlet.go [hunk 1]
```diff
@@ -210,6 +210,10 @@ type LocalSecretsConfig struct {
 	HubConnectionTimeoutSeconds int32 `json:"hubConnectionTimeoutSeconds,omitempty"`
 }
 
+type KubeConfigSecret struct {
+	Name string `json:"name"`
```
Comment (qiujian16) [inline operator/v1/types_klusterlet.go:216]: comment is needed, is this required field?

Comment (xuezhaojun) [reply operator/v1/types_klusterlet.go:216]: Update.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**